src/components/sbb-clock/sbb-clock.tsx

```diff
@@ -89,132 +103,127 @@ export class SbbClock implements ComponentInterface {
 
   private _removeHoursAnimationStyles(): void {
     this._clockHandHours?.classList.remove('sbb-clock__hand-hours--initial-hour');
-    this._element.style.removeProperty('--clock-hours-animation-start-angle');
-    this._element.style.removeProperty('--clock-hours-animation-duration');
+    this._element.style.removeProperty('--sbb-clock-hours-animation-start-angle');
+    this._element.style.removeProperty('--sbb-clock-hours-animation-duration');
   }
 
   private _removeSecondsAnimationStyles(): void {
     this._clockHandSeconds?.classList.remove('sbb-clock__hand-seconds--initial-minute');
     this._clockHandMinutes?.classList.remove('sbb-clock__hand-minutes--no-transition');
-    this._element.style.removeProperty('--clock-seconds-animation-start-angle');
-    this._element.style.removeProperty('--clock-seconds-animation-duration');
+    this._element.style.removeProperty('--sbb-clock-seconds-animation-start-angle');
+    this._element.style.removeProperty('--sbb-clock-seconds-animation-duration');
   }
 
+  /** Given the current date, calculates the hh/mm/ss values and the hh/mm/ss left to the next midnight. */
   private _getCurrentTime(): void {
```

Comment:
Nitpick; This does not actually get the current time, but assign the current time. Should this maybe be renamed to `_assignCurrentTime`?